docs/src/configuration.md

```diff
@@ -108,31 +31,33 @@ The following MPI implementations should work out-of-the-box with MPI.jl:
 - [Fujitsu MPI](https://www.fujitsu.com/global/about/resources/publications/technicalreview/2020-03/article07.html#cap-03)
 - [HPE MPT/HMPT](https://support.hpe.com/hpesc/public/docDisplay?docLocale=en_US&docId=a00105727en_us)
 
-If the implementation is changed, you will need to use [`MPI.use_system_binary()`](@ref MPIPreferences.use_system_binary)
-or [`MPI.use_jll_binary()`](@ref MPIPreferences.use_jll_binary).
+### Configuration
```

Comment:
While not strictly necessary, I propose to make it clear that users need to have MPIPreferences.jl installed separately.

~~~suggestion
Furthermore, you need to install the MPIPreferences.jl package, e.g., by executing
```shell
julia -e 'using Pkg; Pkg.add("MPIPreferences")'
```

### Configuration
~~~

Reply:
Thanks, I added this further up.